build.gradle

```diff
@@ -185,6 +196,7 @@ test {
     dependsOn buildJniLib
     systemProperty 'tests.security.manager', 'false'
     systemProperty "java.library.path", "$rootDir/jni/release"
+    environment('PATH', System.getenv('PATH') + ";$rootDir/jni/release" + ";$rootDir/src/main/resources/windowsDependencies")
```

Comment:
Can we wrap this in platform if/else? Otherwise dll might get added to linux path

Reply:
I think it will not impact the linux build even if it gets added to path. But, still I have added if/else condition